lib/resources/dstu2/encounter.yaml

```diff
@@ -0,0 +1,198 @@
+---
+name: Encounter
+field_name_base_url: http://hl7.org/fhir/DSTU2/encounter-definitions.html#Encounter
+fields:
+- name: identifier
+  required: 'No'
+  cardinality: 0..*
+  type: Identifier
+  description: Identifier(s) by which this encounter is known.
+  children:
+
+  - name: use
+    required: 'No'
+    cardinality: 0..1
+    type: code
+    description: The purpose of the identifier.
+    binding:
```

Comment:
Removed non required.
Only documenting:
Encounter.identifier.type (Extensible)
Encounter.type (Example)
Encounter Priority (Example)
Encounter.participant.type (Extensible)
Encounter.reason (Example) - We also do not support any binding.
Encounter.hospitalization.admitSource (Preferred)
Encounter.hospitalization.dietPreference (Example)
Encounter.hospitalization.specialCourtesy (Preferred)
Encounter.hospitalization.specialArrangement (Preferred)
Encounter.hospitalization.dischargeDisposition (Preferred)

Location.type (Extensible)